Pipfile

```diff
@@ -39,12 +38,13 @@ async_lru = "*"
 geojson = "*"
 orjson = "*"
 cachetools = "*"
-psycopg2 = "*"
-gino = "*"
+psycopg2-binary = "*"
```

Comment:
Presumably because of an error message when attempting to "pipenv update" in OSX.
We don't want the binary version deployed though: https://www.psycopg.org/docs/install.html#psycopg-vs-psycopg-binary
Instead, I believe doing a "brew install libpq" locally should allow you to successfully update the dependencies (possibly with the addition of specifying the path to libpq binaries and/or your Homebrew openssl - search our Slack channel for "pg_config" for a note I wrote recently)